msg.proto

```diff
@@ -99,4 +107,22 @@ message Msg {
   bytes write_file_sync_data = 131;
   uint32 write_file_sync_perm = 132;
   // write_file_sync_perm specified by https://godoc.org/os#FileMode
+
+  // HTTP_CREATE
+  int32 http_server_id = 133;
```

Comment:
For each command type, the prefix of the members should contain the command name. So this should be called `http_create_server_id`. I want to strictly enforce this so keep this sane as it grows.

Furthermore, you should start the indexing at 10 times the index of the command. That is, since `HTTP_CREATE = 14`, this should start at 140.
```
int32 http_create_server_id = 140;
```

Finally, I think HTTP_CREATE is unnecessary. Why not have HTTP_LISTEN response with the id?